tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V4Isolated.AspNetCore/Program.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.Azure.Functions.Worker.Builder;
+using Microsoft.Extensions.Hosting;
+
+var builder = FunctionsApplication.CreateBuilder(args);
+
+builder.ConfigureFunctionsWebApplication();
+builder.UseFunctionExecutionMiddleware();
+builder.UseOutputBindingsMiddleware();
```

Comment:
Do you _have_ to call these? 🤔

If you call this, are they added automatically?

```
var host = new HostBuilder()
    .ConfigureFunctionsWebApplication()
    .Build();
```

Reply:
> But they didn't seem to impact anything tracing wise, so maybe it'd be best to remove them for clarity

I'm happy either way, I was just concerned they were _required_ 🙂